backend/src/modules/azure/applications/register-or-login.azure.use-case.ts

```diff
@@ -75,7 +100,7 @@ export class RegisterOrLoginAzureUseCase implements RegisterOrLoginAzureUseCaseI
 			userToAuthenticate.avatar = avatarUrl;
 		}
 
-		return signIn(userToAuthenticate, this.getTokenService, 'azure');
+		return await signIn(userToAuthenticate, this.getTokenService, 'azure');
```

Comment:
```suggestion
		return signIn(userToAuthenticate, this.getTokenService, 'azure');
```
I think you don't need the `await` here